### PR TITLE
Adds Util class

### PR DESCRIPTION
**Pull request by sagikazarmark**
Adds feature mentioned in #152

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <?php
 
+use Bernard\Util;
 use Bernard\Message;
 
 function bernard_guess_queue(Message $message)
 {
-    return trim(strtolower(preg_replace('/[A-Z]/', '-\\0', $message->getName())), '-');
+    return Util::guessQueue($message);
```
Comment (henrikbjorn) [inline src/functions.php]: Can we remove this function now?

File: src/Util.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Bernard;
```
Comment (henrikbjorn) [inline src/Util.php]: If the goal is testing, we should add some test :)

Comment (sagikazarmark) [inline src/Util.php]: I am writting spec tests ATM. Will write one for this as well.

Comment (henrikbjorn) [inline src/Util.php]: Ohh nice!

File: src/functions.php (hunk 1 above)
Comment (sagikazarmark) [inline src/functions.php]: Only the producer used it. Updated it, and removed the functions.php file.

**Comment by sagikazarmark** [comment]
Can you merge this, so I can write tests for it? Tests go into #158

**Comment by henrikbjorn** [comment]
Thanks

**Comment by sagikazarmark** [comment]
Thanks

**Pull request merged**